cocos/rendering/custom/pipeline.ts

```diff
@@ -1660,7 +1660,6 @@ export interface Pipeline extends BasicPipeline {
 export interface PipelinePassBuilder {
     getConfigOrder (): number;
     getRenderOrder (): number;
-    resetCamera? (cameraConfigs: { [name: string]: any }): void;
```

Comment:
Will removing the public interface `resetCamera` cause compatibility issue?

Reply:
`resetCamera` is newly added in 3.8.5. So remove it will not cause problem

Reply:
Got it.